app/src/main/java/me/ccrama/redditslide/util/LinkUtil.java

```diff
@@ -167,13 +171,30 @@ public static Uri formatURL(String url) {
         }
 
         Uri uri = Uri.parse(url);
-        Uri toReturn;
-        try {
-            toReturn = uri.normalizeScheme();
-        } catch (NoSuchMethodError e) {
-            toReturn = uri;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
I forgot to mention I did this too. Same behavior but does the check without having to try-catch